examples-s3-sdk-extension/src/main/scala/za/co/absa/atum/examples/SampleSdkS3Measurements2.scala

```diff
@@ -89,7 +90,7 @@ object SampleSdkS3Measurements2 {
 
   private def extractCheckpointsRecordCounts(controlMeasure: ControlMeasure): Seq[(String, Int)] = {
     controlMeasure.checkpoints.map { checkpoint =>
-      val count: Int = checkpoint.controls.collectFirst { case Measurement("recordCount", _, _, value) => value.toString.toInt }.getOrElse(0)
+      val count: Int = checkpoint.controls.collectFirst { case Measurement(ControlType.Count.value, _, value) => value.toString.toInt }.getOrElse(0)
```

Comment:
@dk1844 could you please look at this

Reply:
In what way to look at this? In general, I don't mind this change.

_Deep-dive:_
The change is probably based on the fact that the example code expects to find an _INFO file that contains certain counts (such file may be in fact identical or similar to file `examples/data/input/wikidata.csv.info` - this file specifically has the first control count of type `ControlType.Count`). If you have all the prerequisites (locally defined env variables and a valid SAML profile, it is also manually runnable).
In the end, the change specifically decided to assume a control type.

tldr: the example is thus a bit different, but equally valid. 🤷‍♂️

Reply:
I added "*" as controlCol. Taken from wikidata.csv.info.